src/utils.ts

```diff
@@ -249,6 +255,70 @@ const getConstructorParams = (initParams?: Partial<ConstructorParams>) => {
   return p
 }
 
+const createOverlayOnRedirection = () => {
+  const shouldCreateOverlay = checkIfRedirection()
+  if (!shouldCreateOverlay) {
+    return
+  }
```

Comment:
`checkIfRedirection` can be called directly in the if.

example: `if(checkIfRedirection())`

Reply:
Changed name, directly being called now.